### PR TITLE
Feat: Refactor Navigation style to match MVP as per Figma design 

### LINKED ISSUE #117: Feat: Design Navbar
## Description
As a public user,
I need to see a well maintained and designed navbar ,
so that I can navigate properly through the website.

## Mocks
As of the [Design](https://www.figma.com/file/jxCqu1lq55H7jC4FDY119a/BridgeInTech?node-id=10%3A1) , Add the proper design to navbar.
Screenshot (128)

### Update [Required]
- [ ] Avoid changing the other things in this PR.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 day.

### PR DESCRIPTION
**Pull request by prathmeshkhandelwal1**
### Description

NavBar design Done

Fixes #117 

### Type of Change:

- Code
- Quality Assurance
- User Interface

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### Screeshot of change

Screenshot (140)

### CODE REVIEW

**Comment by prathmeshkhandelwal1** [comment]
@mtreacy002  Sorry for the late PR. Actually I was out of town. Plz have a look.

**Comment by mtreacy002** [comment]
That's ok. Thanks, @prathmeshkhandelwal1 for the update. I'll review it today.

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@prathmeshkhandelwal1 , I've tested the navbar and overall it looks good. Although, I am wondering if it's possible for you to fix the positioning of Register/Login or Logout/Welcome back <user>! section when user click main navbar with submenus (note that this bug exist on the original issue, but since the section are separate to the rest of Navbar (left vs right) and not together as the new change, the bug is not as obvious):

**Before the change:**
* when `Home` tab is clicked:
Screen Shot 2020-10-24 at 10 16 02 pm

* when `My Space` tab is clicked:
Screen Shot 2020-10-24 at 10 16 14 pm

**After the change**
* when `Home` tab is clicked:
Screen Shot 2020-10-24 at 9 58 45 pm

* when `My Space` tab is clicked:
Screen Shot 2020-10-24 at 10 00 26 pm

Let me know if you have a question 😉.

**Comment by prathmeshkhandelwal1** [comment]
> @prathmeshkhandelwal1 , I've tested the navbar and overall it looks good. Although, I am wondering if it's possible for you to fix the positioning of Register/Login or Logout/Welcome back ! section when user click main navbar with submenus (note that this bug exist on the original issue, but since the section are separate to the rest of Navbar (left vs right) and not together as the new change, the bug is not as obvious):
> 
> **Before the change:**
> 
> * when `Home` tab is clicked:
> 
> Screen Shot 2020-10-24 at 10 16 02 pm
> 
> * when `My Space` tab is clicked:
> 
> Screen Shot 2020-10-24 at 10 16 14 pm
> 
> **After the change**
> 
> * when `Home` tab is clicked:
> 
> Screen Shot 2020-10-24 at 9 58 45 pm
> 
> * when `My Space` tab is clicked:
> 
> Screen Shot 2020-10-24 at 10 00 26 pm
> 
> Let me know if you have a question 😉.

okk i will try to fix it..as here `accordian` is used from react/bootstrap bcoz of that i was unable to do it. i will try once again maybe i can do something.

**Comment by prathmeshkhandelwal1** [comment]
@mtreacy002 I tried many solns but its now working can you help me with this?

**Comment by prathmeshkhandelwal1** [comment]
@mtreacy002 Can you review it plz?

File: debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[1016/123751.432:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (mtreacy002) [inline debug.log]: Can you please add this file on .gitignore so it won't be included in the code to be merged?

**Review by mtreacy002 — APPROVED** [review]
This PR is tested locally on MacOS Cataline 10.15.7 with Google Chrome browser.
The followings are the test results:
* expected behaviour: Main Nav shows layout as per [proposal design](https://www.figma.com/file/jxCqu1lq55H7jC4FDY119a/BridgeInTech?node-id=10%3A1) in Figma file
* actual behaviour: as expected

**Additional info:**
The bug which shows the Register/Login and Logout/Welome <user>! Navs drop when `Home` and `My Space` tabs were clicked on initial PR has also been removed.
On non-logged in user:
Screen Shot 2020-10-27 at 2 56 52 pm
Screen Shot 2020-10-27 at 2 57 05 pm

On logged-in user:
Screen Shot 2020-10-27 at 2 57 41 pm
Screen Shot 2020-10-27 at 2 57 25 pm

Well done, @prathmeshkhandelwal1  and thank you for your ocntribution 🎉🎉🎉

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
Sorry, I'll reapprove after you remove the debug.log

**Comment by prathmeshkhandelwal1** [comment]
> Sorry, I'll reapprove after you remove the debug.log

Sure

**Review by mtreacy002 — APPROVED** [review]
Thanks, @prathmeshkhandelwal1 for accommodating the change requested. It looks good now 👍

**Comment by prathmeshkhandelwal1** [comment]
> Thanks, @prathmeshkhandelwal1 for accommodating the change requested. It looks good now 👍

😉

**Pull request merged**